src/components/Home.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable camelcase */
```

Comment:
- [OPTIONAL]  Linters are tools that will scan your code with a set of style rules and will report any errors to you that they find, instead of disabling `eslint`, it will be nice to fix the issue.